src/types/api.ts

```diff
@@ -0,0 +1,648 @@
+interface BaseListResponse {
+  meta: object;
+  jsonapi: object;
+  links: object;
+  included?: Array<T>;
+}
+
+interface BaseIndividualResponse {
+  jsonapi: object;
+  links: object;
+  included?: Array<T>;
+}
```

Comment:
Same with this one. Needs a `T` generic added.
```suggestion
interface BaseIndividualResponse<T> {
  jsonapi: object;
  links: object;
  included?: Array<T>;
}
```